test/metabase/request/util_test.clj

```diff
@@ -34,12 +34,12 @@
             :embedded           false
             :ip_address         "0:0:0:0:0:0:0:1"}
            (req.util/device-info @mock-request))))
-  (testing "Embedded request"
+  (testing "SDK request"
```

Comment:
The embedded analytics JS test has been removed since there's no header check for it on 55.